client/README.md

```diff
@@ -192,3 +192,746 @@ For example:
+
+## Future Development
+
+By steps:
+
+Step One: When under 1000px, password requirements box stays the same size - but the input fields change size, are no longer lined up, and are not centered. The text input boxes need to remain the same size, or be centered upon screen resolution shrink.
+
+![Screenshot](Screenshot.jpg)
+
+Step Three: Responsive, but the dream destination text is cut off when screen resolution is 350px or less.
+
+# Login Page
+
+## Description
+
+This is where the user logs in with their username and password
+
+## URL
+
+"/login"
+
+## Current Status
+
+The Login page is fully functional, but not fully responsive.
+
+## Future Development
+
+Text-input fields should not be dynamic and should remain the same size (ideally the same size as the Submit button). Future contributors may consider adding a Back/Cancel button and a Reset Password feature.
+
+There is no front-end or back-end method for a user to reset their password on their own nor by an admin.
+
+# Community Home Page
+
+## Description
+
+This page is where the user can search for specific groups, connect with "other moms", and discover different groups. Also, this page suggests groups for the user to join based on the location the user specifies.
+
+## URL Route
+
+"/community/join-groups"
+
+## Current Status
+
+The page is static and does not pull any data from the backend.
+
+The page needs to make API calls to get group locations (city and state) so that the groups rendered align with the location specified by the user.
+
+This page needs data from the Groups model on the back-end.
+
+## Future Development
+
+1. Make the search bar, "sort by" filter, and "See More" features function.
+2. Pills should render group data pulled from the back-end. When clicked, pills should change color and redirect the user to that group's page.
+
+# Community Groups Page
+
+## Description
+
+After a user clicks on a specific group, they are redirected to this page where they can learn more about a specific group, see different user comments and hashtags, a description of the page, members that are part of the group, and upcoming events.
+
+## URL Route
+
+"/community/view-group"
+
+## Current Status
+
+The page has all necessary components but only renders mock data. The page is mostly mobile responsive. The page needs to fetch group data from the backend to search the group in the group modals.
+
+## Future Development
+
+Different users should be able to like a post and comment on the post, click on the hashtag, click "join" to join the group, click on members to see different members, and click on upcoming events.
+
+# Community People Page
+
+## Description
+
+This page allows the users to view members by location. The user can also search for groups, and filter search results with the sort feature.
+
+## URL Route
+
+"/community/view-group"
```

Comment:
Please put this in a backtick quote, so it looks like this:
`/community/view-group`